### PR TITLE
Use `friendlysize` helper consistently

### PR DESCRIPTION
**Pull request by xPaw**
Pointed out by @fnutt in #3595

### CODE REVIEW

File: test/client/js/helpers/friendlysizeTest.js [hunk 1]
```diff
@@ -5,14 +5,23 @@ const friendlysize = require("../../../../client/js/helpers/friendlysize").defau
...
 
 	it("should round with 1 digit", function() {
-		expect(friendlysize(1234567)).to.equal("1.2 MB");
+		expect(friendlysize(1234567)).to.equal("1.2 MiB");
 	});
 
 	it("should render special case 0 as 0 Bytes", function() {
-		expect(friendlysize(0)).to.equal("0 Bytes");
+		expect(friendlysize(0)).to.equal("0 B");
+	});
```
Comment (NeatNit) [inline test/client/js/helpers/friendlysizeTest.js]: From the title "it should render **special case** 0 as 0 Bytes" it sounds like it was very deliberate and it shouldn't change.

Comment (xPaw) [reply test/client/js/helpers/friendlysizeTest.js]: It's a special case because Math.log(0) doesn't work, there's a `size>0` in the code for that.

Comment (NeatNit) [reply test/client/js/helpers/friendlysizeTest.js]: Oh.

As a user though, I think "Bytes" is probably clearer than "B", so perhaps that shouldn't change.

Comment (xPaw) [reply test/client/js/helpers/friendlysizeTest.js]: That is a good point, I will change that.

**Pull request merged**